pkg/operator/resources/cluster/apiserver.go

```diff
@@ -94,6 +96,35 @@ func getAPIServerClusterPolicyRules() []rbacv1.PolicyRule {
 				"get",
 			},
 		},
+
+		{
+			APIGroups: []string{
+				"",
+			},
+			Resources: []string{
+				"persistentvolumes",
+			},
+			Verbs: []string{
```

Comment:
Maybe I am missing something, I don't see where we are interested in PVs in the web hook. Or is it because renderPvcSpecVolumeModeAndAccessModes calls a function that could need it?

Reply:
right